mlflow/tracking/_model_registry/fluent.py

```diff
@@ -150,7 +150,8 @@ def _register_model(
         runs_artifact_repo = RunsArtifactRepository(model_uri)
         # List artifacts in `<run_artifact_root>/<artifact_path>` to see if the run has artifacts.
         # If so use the run's artifact location as source.
-        if runs_artifact_repo._is_directory(""):
+        artifacts = runs_artifact_repo._list_run_artifacts(path=None)
+        if "MLmodel" in (art.path for art in artifacts):
```

Comment:
Is there a constant we can use? Do we have coverage for all the relevant cases here (registration from run, registration from model)?

Reply:
Yes, I'll de-hardcode this.